application/extensions/admin/survey/ListSurveysWidget/views/listSurveys.php

```diff
@@ -49,8 +50,7 @@
                 [
                     'header'            => gT('Survey ID'),
                     'name'              => 'survey_id',
-                    'type'              => 'raw',
-                    'value'             => 'CHtml::link($data->sid, Yii::app()->createUrl("surveyAdministration/view/",array("iSurveyID"=>$data->sid)))',
```

Comment:
Maybe I'm an idiot, but how is removing links related to XSS? o0

Reply:
type text is running the value through encoding, we tried to get a lot of information into that field by adding raw type. So instead we now have an option for the gridview that can add this functionality